src/app-logic/url-handling.js

```diff
@@ -722,7 +722,7 @@ export function upgradeLocationToCurrentVersion(
 /* eslint-disable no-useless-computed-key */
 const _upgraders = {
   [0]: (processedLocation: ProcessedLocationBeforeUpgrade, _: Profile) => {
-    // Version 1 is the first versioned url.
+    // Version 0 is the first versioned url.
```

Comment:
Actually the previous comment is right: before v=1 the URL wasn't versioned explicitely, there was no `v=0` in it. We were doing some adhoc conversion (that we kept in the step for "0"). Probably the comment can be rewritten a bit to be clearer and explain this history better.

Reply:
Hmm, then I don't really understand what's going on. If v=1 was the first version, why wasn't the [0] upgrader code inside the [1] upgrader?

Reply:
I'll look at the history of this file, maybe I can answer this question myself.

Reply:
Yeah upgraders [0] and [1] were added at the same time. I will move the [0] code into [1] because the two are always supposed to run at the same time.

Reply:
I've made that change, re-requesting review.

Reply:
Yeah this makes sense, I think at the time the author wanted to be explicit about what their patch was doing (and so having a specific upgrade step for their patch)